app/views/comments/create.js.erb

```diff
@@ -0,0 +1,11 @@
+let added_comment = $("<%= escape_javascript(render 'comments/comment', comment: @comment) %>");
+
+added_comment.hide();
```

Comment:
I'm not sure if this is why it's happening, but when I try to create a comment right after already posting another, I'm not able to.

Reply:
Nice catch, Amanda! Yes, this needs to be addressed.